libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1025,12 +1027,15 @@ void AP_AHRS_DCM::estimate_wind(void)
         }
 
         _last_wind_time = now;
-    } else if (now - _last_wind_time > 2000 && airspeed_sensor_enabled()) {
+    }
+#if AP_AIRSPEED_ENABLED
+     else if (now - _last_wind_time > 2000 && airspeed_sensor_enabled()) {
```

Comment:
Suggest doing an early-return in this function to do without the `else`

Reply:
Done. Leaving open for you to check.